### DISCUSSION TITLE
Documentation isn't correctly display

### DISCUSSION THREAD

**Discussion by VisionaryAppDev** [mfussenegger/nvim-jdtls#332] [Q&A] [answered]
Hello everyone,

I think document isn't display correctly when using JDTLS and not sure what's wrong. It's always containing something like `jdt://contents/....`. I have tried using other language server and it's working just fine. Please check the below screenshot for what I mean.

 
![Screenshot_20220927_135443-1](https://user-images.githubusercontent.com/27724011/192455043-a9d9569c-a424-4f9f-b5cb-5c4da6ec7953.png)
![Screenshot_20220927_135400](https://user-images.githubusercontent.com/27724011/192455042-eae80407-c894-4240-a869-d063580d842d.png)

This last one is from other language server
![Screenshot_20220927_135509](https://user-images.githubusercontent.com/27724011/192455051-eb1871b8-6d4f-4983-ac11-61ef074b787b.png)

So, does anyone know how to get it fixed plz?

**Comment by mfussenegger** [comment, marked as answer]
See https://github.com/neovim/neovim/issues/20246
